EndpointHandler.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+require "settings.php";
+
+// Decodes the data array and returns the value of "claim"
+function getClaim($claim, $data) {
+	
+	$data_array = json_decode($data, true);
+	return $data_array[$claim];
+}
```

Comment:
Also, this mechanism is causing you to call `json_decode` every single time you want to get a claim. It would probably be much more efficient to decode the JSON once, and then just grab the claim needed.